service/routes.py

```diff
@@ -510,6 +534,64 @@ def get(self, shopcart_id, item_id):
         item_js = item.serialize()
         return item_js, status.HTTP_200_OK
 
+        @api.doc("put_items")
```

Comment:
could you help remove the extra tab here? unit tests seem to fail because of this lol

Reply:
how about aligning the naming style, e.g. use `@api.doc("update_items")` instead of `@api.doc("put_items")`